cpp/src/slice2cpp/Gen.cpp

```diff
@@ -4339,6 +4389,7 @@ Slice::Gen::ImplVisitor::ImplVisitor(Output& h, Output& c, const string& dllExpo
 {
 }
 
+// TODO remove ImplVisitor and replace it with the Cpp11ImplVisitor.
```

Comment:
It's actually the other way around - rename the Cpp11Xxx into the non-Cpp11 functions/classes.

Reply:
Yeah, that's what I was trying to say... I'll rewrite it, but so it actually makes sense!